ui/app/src/views/CreatePoolPage.vue

```diff
@@ -132,11 +134,18 @@ export default defineComponent({
             return "Please enter an amount";
           case PoolState.INSUFFICIENT_FUNDS:
             return "Insufficient Funds";
+          case PoolState.ZERO_ROWAN: 
```

Comment:
Maybe I am missing context here but I am not sure this logic is correct.

We should make the distinction between adding and creating a pool by looking at the pools that exist and if a pool exists we should be adding liquidity or if the pool doesn't exist we are creating a pool.